cardinal_toe_app/src/app/services/number-randomiser/number-randomiser.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NumberRandomiserService {
+
+      /**
+  * Returns a pseudorandom `number` between `0` and `max`.
+  * @function
+  * @param {number} max `REQUIRED`
+  */
+      getRandomNumber = (max: number): number => Math.floor(Math.random() * max);
```

Comment:
this will never reach `max` because floor always rounds down

Reply:
Of course, will adjust.